R/uvhydrograph-fnc.R

```diff
@@ -65,13 +86,56 @@ add_working_dv <- function(times, points){
   type = 'p'
 }
 
+add_third_axes <- function(secondary_lims = NULL, tertiary_pts = NULL, tertiary_lbl = NULL, measured_shift_pts = NULL) {
+  if(!is.null(tertiary_pts)) {
+    par(new = TRUE)
+    
+    lims <- getUvhLims(tertiary_pts)
+    xaxis <- lims$xlim
+    yaxis <- lims$ylim
+    
+    #expand y limits if error bars bleed over
+    if(!is.null(measured_shift_pts)) {
+      errorBarLims <- getUvhLims(measured_shift_pts, yMinField = "minShift", yMaxField = "maxShift")
+      combinedLims <- combineLims(lims, errorBarLims)
+      yaxis = combinedLims$ylim
```

Comment:
I would like to come up with a better way to resize lims after the data are added in, instead of having to manually check lims